minikube_demo_setup.sh

```diff
@@ -61,7 +61,10 @@ function print_min_resources() {
 
 # Checks if the system which tries to run autotune is having minimum resources required
 function sys_cpu_mem_check() {
-	SYS_CPU=$(lscpu -p | grep -v '^#' | wc -l)
+	cpuinfo=/proc/cpuinfo
+	core=$(awk '/^cpu cores/{print $4; exit}' <"$cpuinfo" )
+	siblings=$(awk '/^siblings/{print $3; exit}' <"$cpuinfo")
+	SYS_CPU=$((core * siblings))
```

Comment:
This calculates incorrectly on my laptop. It shows SYS_CPU as `32` (4 cores * 8 siblings) whereas I have only 8 processors total. I would suggest a simple calculation as below should do it
```
SYS_CPU=$(cat /proc/cpuinfo | grep "^processor" | wc -l)
```

Reply:
ok, i misunderstood SYS_CPU=number of core in total. But, SYS_CPU is number of processors. I will make the changes as mentioned above. Thank you.